README.md

```diff
@@ -113,3 +113,11 @@ yarn unlink @polkadotcloud/dashboard-ui-dev
 ```
 
 Followed by `yarn unlink` in The Package project directory.
+
+### Unregister
```

Comment:
This is unlink no? Already mentioned above

Reply:
unlinking is unlinking the package, but the package will still be in your computer environment, it's just not in your project